namespaces/live-1.cloud-platform.service.justice.gov.uk/parliamentary-questions-staging/resources/elasticache.tf

```diff
@@ -0,0 +1,36 @@
+################################################################################
+# Parliamentary Questions
+# Application Elasticache for Redis (for Sidekiq background job processing)
+#################################################################################
+
+module "parliamentary_questions_elasticache_redis" {
+  source                 = "github.com/ministryofjustice/cloud-platform-terraform-elasticache-cluster?ref=5.1"
+  cluster_name           = var.cluster_name
+  team_name              = var.team_name
+  business-unit          = "DEX uhura"
```

Comment:
Can the business-unit be one of the values mentioned in here: https://github.com/ministryofjustice/cloud-platform-cli/blob/main/pkg/environment/businessUnitValidator.go#L9-L15. Or is it a different business unit?

Reply:
Need to decide whether the BU list needs to be expanded to add DEX